lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -228,6 +230,19 @@ def test_is_past_due_no_graceperiod(self):
         self.xblock.due = now + timedelta(days=1)
         self.assertFalse(self.xblock.is_past_due)
 
+    @patch('lti_consumer.lti_consumer.timezone.now')
```

Comment:
`@patch('lti_consumer.lti_consumer.timezone.now')` has the same effect, but just for the lti_consumer.lti_consumer module.